conference.js

```diff
@@ -1916,6 +1897,14 @@ export default {
             JitsiConferenceEvents.PARTICIPANT_PROPERTY_CHANGED,
             (participant, name, oldValue, newValue) => {
                 switch (name) {
+                case 'features_screen-sharing': {
+                    APP.store.dispatch(participantUpdated({
+                        conference: room,
+                        id: participant.getId(),
+                        features: { 'screen-sharing': true }
```

Comment:
Does this override an existing features object in the redux participant or does it update the key "screen-sharing" in the features object? This is the case I'm thinking about:
1. User A update comes in for feature Foo.
2. The redux stuff happens so the User A in redux has features `{ Foo: true }`.
3. This participantUpdated is fired.
4. The redux stuff happens so the User A in redux has features `{ screen-sharing: true }` and no longer has Foo. 

Is that the case?

Reply:
Well, we have no other features at the moment. And probably won't have. This is a quick workaround discussed offline for screen sharing.
What you say that we should be merging these features not overriding them?

Reply:
I can remove this features object and directly set screen-sharing-enabled for example in the participant, if that sounds ok?

Reply:
I noticed "outbound-call" being used in the AddPeopleDialog; it looked like it was a feature on the local participant object. I'm okay with the current implementation if you're certain there are no other features.

Reply:
Well, that is for the local participant ... while I think this participantUpdated is for the remotes. I will check it and maybe to ask for some help about this redux/state stuff. Thanks for noticing.

Reply:
And the only feature we advertise is this:
```
+    if (features && features['screen-sharing'] === 'true') {
+        conference.setLocalParticipantProperty('features_screen-sharing', true);
+    }
```
So at first quick look it seems ok.

Reply:
But yeah I need to double check is that PARTICIPANT_PROPERTY_CHANGED fired for local participant ...

Reply:
I see. You're right. PARTICIPANT_PROPERTY_CHANGED gets fired from the lib's JitsiParticipant, and JitsiParticipant instances are created only for remote users,

Reply:
I don't see it got fired for the local participant.